app/views/ips/_confirm_rotate_radius_key.html.erb

```diff
@@ -17,5 +17,8 @@
 
     <%= button_to "Yes, rotate this RADIUS key", location_path(@key_to_rotate),
```

Comment:
Hi Mel, this doesn't read continuously, what I hear is "Yes, rotate this key", then it stops, if I press down I then here "for <address>", feels a bit jarring, and this is because the 'span' is written outside of the button, and indeed form, which you can see if you look at the html output. 
I don't think 'button_to' has any aria features, so you may have to 'hand craft' the form and button like.
```
  <%= form_with(url: @key_to_rotate, method: :patch) do |form| %>
    <div>
      <button class="govuk-button red-button" type="submit" aria-label="Yes, rotate  RADIUS key for <%= @key_to_rotate.address %>"> Yes, rotate this RADIUS key</button>
    </div>
  <% end %>
```